### PR TITLE
Add maintainer to nfpm config for goreleaser

### PR DESCRIPTION
**Pull request by itzg**
# Resolves

https://jira.rax.io/browse/SALUS-443

# What

Adds the maintainer to the deb/rpm package produced by goreleaser.

### CODE REVIEW

File: .goreleaser.yml [hunk 1]
```diff
@@ -33,6 +33,7 @@ nfpm:
   homepage: https://www.rackspace.com/
   description: Monitoring application which submits system metrics for further analysis.
   license: Apache 2.0
+  maintainer: monitoring@rackspace.com
```
Comment (jjbuchan) [inline .goreleaser.yml]: Who gets emails to that?

Comment (itzg) [reply .goreleaser.yml]: I don't know, I copied it from our agent package:

```
$ dpkg -s rackspace-monitoring-agent
dpkg-query: warning: parsing file '/var/lib/dpkg/status' near line 5375 package 'telemetry-envoy':
 missing maintainer
Package: rackspace-monitoring-agent
Status: install ok installed
Priority: optional
Section: devel
Installed-Size: 4930
Maintainer: monitoring@rackspace.com
Architecture: amd64
Version: 2.6.15
Description: Rackspace Monitoring Agent
```

Comment (jjbuchan) [reply .goreleaser.yml]: I emailed it and have heard nothing back.  We should probably pick something else or talk to help desk to see if we can get control of it.

Comment (itzg) [reply .goreleaser.yml]: Ticket has been submitted

**Comment by itzg** [comment]
We have regained control of monitoring@rackspace.com

**Pull request merged**